watertap/unit_models/ion_exchange_0D.py

```diff
@@ -544,594 +601,688 @@ def build(self):
-        )
+            self.num_traps = 5  # TODO: make CONFIG option
+            self.trap_disc = range(self.num_traps + 1)
+            self.trap_index = self.trap_disc[1:]
 
-        # ====== Rinse ====== #
+            self.c_trap_min = Param(  # TODO: make CONFIG option
+                initialize=0.01,
+                mutable=True,
+                doc="Minimum relative breakthrough concentration for estimating area under curve",
+            )
 
-        self.rinse_bv = Var(
-            initialize=5,
-            bounds=(2, 10),
-            doc="Number of bed volumes for rinse step",
-        )
+            # TODO: use trapezoidal approach for langmuir?
+            self.c_traps = Var(
+                self.trap_disc,
+                initialize=0.5,
+                bounds=(0, 1),
+                units=pyunits.dimensionless,
+                doc="Normalized breakthrough concentrations for estimating area under breakthrough curve",
+            )
 
-        self.rinse_flow = Var(
-            initialize=0.1,
-            units=pyunits.m**3 / pyunits.s,
-            doc="Rinse volumetric flow rate",
-        )
+            self.tb_traps = Var(
+                self.trap_disc,
+                initialize=1e6,
+                bounds=(0, None),
+                units=pyunits.second,
+                doc="Breakthrough times for estimating area under breakthrough curve",
+            )
 
-        self.t_rinse = Var(
-            initialize=360, units=pyunits.s, bounds=(120, None), doc="Rinse time"
-        )
+            self.traps = Var(
+                self.trap_index,
+                initialize=0.01,
+                bounds=(0, 1),
+                units=pyunits.dimensionless,
+                doc="Trapezoid areas for estimating area under breakthrough curve",
+            )
 
-        self.main_pump_power = Var(
-            initialize=0.1, units=pyunits.kW, doc="Main pump power"
-        )
+            self.c_traps[0].fix(0)
+            self.tb_traps[0].fix(0)
 
-        self.regen_pump_power = Var(
-            initialize=0.1, units=pyunits.kW, doc="Regen pump power"
-        )
+            self.c_norm_avg = Var(
+                self.target_ion_set,
+                initialize=0.25,
+                bounds=(0, 2),
+                units=pyunits.dimensionless,
+                doc="Sum of trapezoid areas",
+            )
 
-        self.bw_pump_power = Var(
-            initialize=0.1, units=pyunits.kW, doc="Backwash pump power"
-        )
+            self.c_breakthru = Var(
+                self.target_ion_set,
+                initialize=0.5,
+                bounds=(0, None),
+                units=pyunits.kg / pyunits.m**3,
+                doc="Breakthrough concentration of target ion",
+            )
 
-        self.rinse_pump_power = Var(
-            initialize=0.1, units=pyunits.kW, doc="Rinse pump power"
-        )
+            self.freundlich_n = Var(
```

Comment:
You and I should probably be explicit here, at least in the doc string for n vs 1/n and 'exponent'

Reply:
Yes, I took guidance from GAC here because it is _not_ inverted ``n``